src/figuresClasses.ts

```diff
@@ -1,19 +1,103 @@
 export interface Figure {
+  shape: string;
```

Comment:
```suggestion
  shape: 'triangle' | 'circle' | 'rectangle'
```
Also write union type for color